### PR TITLE
Return company employees in users field

### PR DESCRIPTION
**Pull request by malteo**

### CODE REVIEW

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java [hunk 1]
```diff
@@ -59,7 +59,10 @@
     @Valid
     private BankAccountDataDTO bankAccountData;
 
-    @ApiModelProperty( "Vendors of the company" )
+    @ApiModelProperty( "Users of the company" )
```
Comment (gionn) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java]: annotate with FieldApi

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java [hunk 2]
```diff
@@ -59,7 +59,10 @@
...
+    private List<UrlEntityDTO> users;
+
+    @ApiModelProperty( value = "Vendors of the company", hidden = true )
```
Comment (gionn) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java]: annotate with UnsupportedAPI

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java [hunk 3]
```diff
@@ -206,11 +214,29 @@ public void setNotificationSecretToken( String notificationSecretToken )
...
     public List<UrlEntityDTO> getVendors()
     {
         return vendors;
     }
 
+    /**
+     * @deprecated by {@link #setUsers(List)}
+     */
+    @Deprecated
     public void setVendors( List<UrlEntityDTO> vendors )
```
Comment (AndreaVecchi) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java [hunk 4]
```diff
@@ -206,11 +214,29 @@ public void setNotificationSecretToken( String notificationSecretToken )
...
+    public void setUsers( List<UrlEntityDTO> users )
+    {
+        this.users = users;
+    }
+
+    /**
+     * @deprecated by {@link #getUsers()}
+     */
+    @Deprecated
     public List<UrlEntityDTO> getVendors()
```
Comment (AndreaVecchi) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CompanyDTO.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 2 issues
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**